### PR TITLE
Scripts to update gh-pages

### PR DESCRIPTION
**Pull request by HarshKhandeparkar**
Fixes #0000  (<=== Replace `0000` with the Issue Number)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm run test-all`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] ask `@publiclab/is-reviewers` for help, in a comment below
* [ ] Insert-step functionality is working correct as expected.
> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software
Please make sure to get at least two reviews before asking for merging the PR as that would make the PR more reliable on our part
Thanks!

### CODE REVIEW

**Comment by HarshKhandeparkar** [comment]
I haven't tested it yet though. I changed a few things in it.

**Comment by HarshKhandeparkar** [comment]
@publiclab/is-reviewers I just changed the script, split it into two parts. One script is for manually updating gh-pages and the other is for use in a GitHub action and is also used in the manual script. I also added docs for the same :)

**Comment by HarshKhandeparkar** [comment]
I tested it on my [fork](https://harshkhandeparkar.github.io) and it completed w/o any errors.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -44,6 +44,7 @@
 /*.lock @ubliclab/is-maintainers
 /Gruntfile.js @publiclab/is-maintainers
 /.github/ @publiclab/is-maintainers
+/scripts/ @publiclab/is-maintainers
```
Comment (jywarren) [inline .github/CODEOWNERS:47]: nice!

**Review by jywarren — APPROVED** [review]

**Comment by jywarren** [comment]
Trying this out now to publish to our own main demo!

**Comment by jywarren** [comment]
Just used this! It worked!!!

**Comment by jywarren** [comment]
awesomeeeeeeee.....

**Pull request merged**